s127/admin.py

```diff
@@ -78,91 +84,115 @@ class FeatureTypePermissionTypeInline(nested_admin.NestedGenericTabularInline):
 # endregion Inlines
 
 
-class AccumulatedInlines:
-    def get_inlines(self, *args, **kwargs):
-        accumulated_inlines = []
-        for ancestor_class in type(self).mro():
-            accumulated_inlines.extend(getattr(ancestor_class, "inlines", []))
-        return list(dict.fromkeys(accumulated_inlines).keys())
-
-
 # region InformationTypeAdmins
 
 
-class InformationTypeAdmin(
-    AccumulatedInlines, ModelAdminWithOrderedFormsets, nested_admin.NestedModelAdmin
-):
+class InformationTypeAdmin(nested_admin.NestedModelAdmin):
     inlines = [InformationInline]
```

Comment:
Leaving `inlines` here should trigger an `ImproperlyConfigured` or be unused because we've removed `AccumulatedInlines`.

Reply:
Same for all the `FeatureTypeAdmin` class hierarchy.